service/src/main/java/com/myhome/services/springdatajpa/AuthenticationSDJpaService.java

```diff
@@ -10,26 +10,36 @@
 import com.myhome.services.AuthenticationService;
 import java.time.Duration;
 import java.time.LocalDateTime;
-import lombok.RequiredArgsConstructor;
 import lombok.extern.slf4j.Slf4j;
 import org.springframework.beans.factory.annotation.Value;
 import org.springframework.security.crypto.password.PasswordEncoder;
 import org.springframework.stereotype.Service;
 
 @Slf4j
-@RequiredArgsConstructor
 @Service
 public class AuthenticationSDJpaService implements AuthenticationService {
 
   @Value("${token.expiration_time}")
```

Comment:
If you added the `@Value` annotation in the constructor, then it doesn't need to be on the variable definition as well.